src/aol-hb.js

```diff
@@ -27,4 +32,27 @@ globalContext.init = (bidRequestConfig, placementsConfigs) => {
   };
 };
 
+globalContext.que.push = (cmd) => {
```

Comment:
Are you overriding array method here? Because I don't think this implementation will add the command to the array.